src/seedu/addressbook/parser/Parser.java

```diff
@@ -238,7 +238,7 @@ private int parseArgsAsDisplayedIndex(String args) throws ParseException, Number
      * @return the prepared command
      */
     private Command prepareFind(String args) {
-        final Matcher matcher = KEYWORDS_ARGS_FORMAT.matcher(args.trim());
+        final Matcher matcher = KEYWORDS_ARGS_FORMAT.matcher(args.toLowerCase().trim());
```

Comment:
`Parser` class' responsibility is to parse the arguments. Whose responsibility is it to do the arguments processing?